Sources/Confidence/Confidence.swift

```diff
@@ -339,6 +339,10 @@ extension Confidence {
             return self
         }
 
+        /**
+         Sets the timeout for the network requests to the Confidence backend.
+         The default is 10 seconds.
+         */
         public func withTimeout(timeout: Double) -> Builder {
```

Comment:
can we have something like this maybe:
```
public func withTimeout(seconds/milliseconds/nanoseconds timeout: Double)
```

Reply:
sure!

Reply:
The timeout would be in seconds though, since that's what TimeInterval is using

Reply:
as long as we state in the api what we expect, it's fine for me.